.github/workflows/main.yml

```diff
@@ -1,9 +1,9 @@
 name: Main workflow
 on:
   push:
-    branches: [ 'main' ]
+    branches: ['main']
```

Comment:
In frontend-actions-pilot you added spaces (in the telia-fe import PR). I think I prefer with spaces tbh.

The quotes are unnecessary, but I don't mind keeping them "just in case" since they could be required for some values.